cmd/containerd-release/build.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"runtime"
+)
+
+func build() error {
+	out, err := exec.Command("make").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("%s: %s", err, out)
```

Comment:
doesn't this need to be `string(out)`?

Reply:
no, do you even Go?

Reply:
%s automagically makes it a string representation of the bytes in all the fmt functions

Reply:
Haha, my bad :). I guess I got burn by `%v` a few times and didn't pay attention